packages/discord-request/test/routes.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from "vitest";
+import { getRouteInformation } from "../src/util/routes.js";
+
+test("getRouteInformation should extract snowflakes", () => {
+	expect(
+		getRouteInformation(
+			"/guilds/815369174096412692/channels/905838550522671144"
+		)
+	).toEqual({
+		path: "/guilds/:id/channels/:id",
+		identifier: "815369174096412692",
+	});
+});
+
+test("getRouteInformation should replace snowflakes with `:id`", () => {
```

Comment:
This reminds me, tokens in urls technically count as a different "route" with this. Its kinda vague in the docs, however they're always specified as /route/something/:id/:token/whatever elsewhere in the docs.
Thoughts?

For webhooks they can be considered part of the major id, but interaction callback is an unknown for example.

Reply:
Your interpretation is correct, yeah. In that case, it should be keyed off interaction token

Reply:
Per Discord discussion we handle this correctly right now, but we can tweak the routes definition to extract it as the primary ID to be more "correct"